src/csharp/Grpc.Core/Internal/NativeMetadataCredentialsPlugin.cs

```diff
@@ -80,7 +77,7 @@ private void NativeMetadataInterceptorHandler(IntPtr statePtr, IntPtr serviceUrl
             try
             {
                 var metadata = new Metadata();
-                await interceptor(context, metadata).ConfigureAwait(false);
+                await interceptor(context, metadata);
```

Comment:
what's the point of removing the ConfigureAwait(false)?

Reply:
No big reason, it seemed like this might have been helping to run the continuation on a different thread before, and I couldn't see the use of it now.